analyses/interaction-plots/scripts/03-plot_interactions.R

```diff
@@ -132,3 +158,56 @@ cooccur_plot <- ggplot(
   )
 
 ggsave(cooccur_plot, filename = plot_file)
+
+# if we don't have a disease table, quit 
+if (is.na(opts$disease_table)) {
+ quit()
+}
+# otherwise make a gene by disease stacked bar chart
+
+disease_file = opts$disease_table
+disease_df <-
+  readr::read_tsv(disease_file, col_types = readr::cols()) %>%
+  dplyr::mutate(gene = factor(gene, levels = genes))
+
+
+display_diseases <- disease_df %>%
+  dplyr::group_by(disease) %>%
+  dplyr::tally(wt = mutant_samples) %>%
+  dplyr::arrange(desc(n)) %>%
+  head(7) %>% # seven so we end up with 8 total for color reasons
+  dplyr::pull(disease)
+
+disease_df <- disease_df %>%
+  dplyr::mutate(disease_factor = 
+           forcats::fct_other(disease, keep = display_diseases) %>%
+           forcats::fct_relevel(display_diseases)
+  )
+
+okabe_ito <- c("#E69F00", "#56B4E9", "#009E73", "#F0E442",
+               "#0072B2", "#D55E00", "#CC79A7", "#444444")
+
+disease_plot <- ggplot(
+  disease_df,
+  aes(x = gene, y = mutant_samples, fill = disease_factor)) + 
+  geom_col() +
+  labs(
+    x = "",
+    y = "Samples with mutations",
+    fill = "Diagnosis"
+  ) + 
+  scale_fill_manual(values = okabe_ito) + 
```

Comment:
I think you can do:
```suggestion
  colorblindr::scale_fill_OkabeIto() + 
```
and remove lines 187-188

Reply:
Indeed I can. I just today saw that `colorblindr` is already installed; I was avoiding it to prevent having to `install::github()`, but that is moot!